opentracing-api/src/main/java/io/opentracing/Tracer.java

```diff
@@ -155,46 +160,47 @@
         SpanBuilder withStartTimestamp(long microseconds);
 
         /**
-         * Returns a newly started and activated {@link ActiveSpan}.
+         * Returns a newly started and activated {@link Scope}.
          *
          * <p>
-         * The returned {@link ActiveSpan} supports try-with-resources. For example:
+         * The returned {@link Scope} supports try-with-resources. For example:
          * <pre><code>
-         *     try (ActiveSpan span = tracer.buildSpan("...").startActive()) {
+         *     try (Scope scope = tracer.buildSpan("...").startActive()) {
          *         // (Do work)
-         *         span.setTag( ... );  // etc, etc
-         *     }  // Span finishes automatically unless deferred via {@link ActiveSpan#capture}
+         *         scope.span().setTag( ... );  // etc, etc
+         *     }  // XXX Span finishes automatically
```

Comment:
Should say something like "Span will automatically be finished when the Scope is closed, as the default 'finishOnClose' behavior was chosen when starting the active span"???